azure-iot-device/tests/common/pipeline_test.py

```diff
@@ -0,0 +1,118 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
```

Comment:
>information [](start = 10, length = 11)

I'm not sure why the files in this directory are showing up in your commit.  They already exist in master.

Reply:
shoot.  I messed up.  This is not in master yet.  I'm not sure how I did that, but I'll get it in there asap.

---
In reply to: [288537437](https://github.com/Azure/azure-iot-sdk-python-preview/pull/97#discussion_r288537437) [](ancestors = 288537437)